docs/Conceptual/00 - Introduction.md

```diff
@@ -0,0 +1,188 @@
+
+The ultimate form of this isolated infrastructure is “bring your own service”. In this case, a multi-tenanted provider allows you to configure their solution with your own instances of particular storage or compute infrastructure, encryption keys, credentials etc.
+
+For example, a tenant might configure the text-message dispatch service provided by its parent to use the tenant’s own Twilio account, giving it access to a key in KeyVault that contains their Twilio credentials, configured with access rights appropriate for the service.
+
+## Corvus.Tenancy and Marain.Tenancy
+To help with implementing multi-tenanted solutions using the dotnet platform, we maintain two open-source projects: [Corvus.Tenancy]( https://github.com/corvus-dotnet/Corvus.Tenancy) and [Marain.Tenancy](https://github.com/marain-dotnet/Marain.Tenancy).
+
+### Management plane
+In Corvus.Tenancy, we provide low-level services for creating and managing Tenants as logical entities within your multi-tenanted solution. It also gives individual tenants within that solution the ability to manage features of their own tenancy and tenanted-service configuration.
+
+Marain.Tenancy then exposes these low-level services as APIs for you to develop your own multi-tenanted solutions “as a service”.
+You would use these APIs to develop your own management plane for your multi-tenanted solution.
+
+We also provide tooling and recommended practices to assist in the deployment and management of multi-tenanted solutions using this model. These are found in [Marain.Instance](https://github.com/marain-dotnet/Marain.Instance).
+
+### Solution plane
+Corvus.Tenancy also provides libraries for popular storage, identity, secret management, and compute solutions, like Azure Storage, Cosmos DB, SQL Server and Key Vault. It leverages conventions over tenancy configuration to create appropriate instances of the types in the client SDKs for those technologies, pre-configured for your application’s tenant.
+
+## Summary
+In this document, we’ve looked at aspects of multi-tenanted solution architecture. We talked about
+- multi-tenanted v. multi-user solutions
+- the isolation goals of a multi-tenanted model
+- when to choose a multi-tenanted architecture
+- the hierarchy of multi-tenanted solutions in multi-tenanted solutions
+- separation of the management plane and the solution plane
+- data and compute segregation strategies, including shared infrastructure, isolated infrastructure, and “bring your own service”
+- how Corvus.Tenancy and Marain.Tenancy help with building multi-tenanted services.
+
+## Next steps
+•	[Listomatic– an overview of a multi-tenanted reference solution using Marain.Tenancy](https://github.com/marain-dotnet/Marain.List-O-Matic)
```

Comment:
What's this? (Or at least, if it weren't a 404, what would it be?)

Reply:
I am thinking we need a reference application that shows how all this works.